.github/workflows/test.yml

```diff
@@ -40,43 +40,53 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile --ignore-scripts
       - name: Check Formatting
-        run: yarn prettier -c pages src
+        run: yarn prettier -c .
 
-  test:
+  types:
     runs-on: ubuntu-latest
     needs: lint
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - uses: dorny/paths-filter@v2
-        id: changes
+      - name: Install Node.js
+        uses: actions/setup-node@v3
         with:
-          filters: |
-            server:
-              - 'src/server/**'
-            scripts:
-              - 'src/scripts/**'
-            assets:
-              - 'assets/**'
+          node-version: 16
+          cache: yarn
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile --ignore-scripts
+      - name: Check Types
+        run: yarn test:types
+
+  test:
+    runs-on: ubuntu-latest
+    needs: types
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Get Date
+        id: get-date
+        run: echo "::set-output name=date::$(/bin/date -u "+%Y%m%d")"
```

Comment:
should be updated to use [environment files](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files)